rpc/backend/backend.go

```diff
@@ -160,9 +152,12 @@ func NewBackend(
 		panic(err)
 	}
 
+	rpcClient := clientCtx.Client.(tmrpcclient.Client)
```

Comment:
check if it is a valid client
```suggestion
	rpcClient, ok := clientCtx.Client.(tmrpcclient.Client)
	if !ok {
		return nil, errors.New("invalid rpc client")
	}
```

Reply:
It should panic because if not then it's not properly configured. 

We need to do this conversion bc the SDK uses ABCIClient instead of the full Client interface from Comet rpc.